NEWS.md

```diff
@@ -6,6 +6,10 @@
 * Added missing commas and addressed formatting issues throughout the vignettes and articles. Backticks for package names were removed and missing parentheses for functions were added (@Joscelinrocha, #218).
 
 * Increased the minimum R version to R 4.1.
+
+* Transitioned support for parallel processing fully to the 
+  [future](https://www.futureverse.org/) framework. Parallelism backends
+  registered with foreach will be ignored (#234).
```

Comment:
Is there an easy way to throw a warning? For tune, we've set the stage for the transition over a year. This might be an unexpected surprise for folks.

Reply:
Yes, we could easily throw a warning here with the same logic that tune used! The fact that tune has used that logic for a while, though, is my reasoning for not doing so here, as people can't use stacks without using tune; they'll see the warning from that package before they see this one.